src/cli.rs

```diff
@@ -32,4 +44,5 @@ pub(crate) fn build() -> Command {
         .subcommand(completions::subcommand())
         .subcommand(health::subcommand())
         .subcommand(Command::new("build-info").about("Show build information for installation"))
+        .subcommand(Command::new("open").about("Open interactive TUI"))
```

Comment:
I would not have expected "open" to open a tui. Perhaps calling it `ui` or `tui` might make more sense?

Reply:
if you prefer `open` I'm fine with that.

The fact that cypress does is a logical fallacy however